### PR TITLE
add article about deleting a group of builds using one command

### PR DESCRIPTION
**Pull request by dturecek**
This is a work in rogress, waiting for Copr release.

### CODE REVIEW

**Comment by xsuchy** [comment]
@praiskup @FrostyX please review after my change.

File: _posts/2020-06-01-deleting-list-of-builds.md [hunk 1]
```diff
@@ -4,8 +4,16 @@ author: Dominik Tureček
 layout: post
 ---
 
-Since the latest release, we are now able to delete a group of builds specified by their IDs with a single command in the following style:
+Previously, when you wanted to delete lots of build in the Copr you have to do something like:
```
Comment (FrostyX) [inline _posts/2020-06-01-deleting-list-of-builds.md:7]: Previously, when you wanted to delete lots of build in the Copr, you had to do something like:

File: _posts/2020-06-01-deleting-list-of-builds.md [hunk 2]
```diff
@@ -4,8 +4,16 @@ author: Dominik Tureček
...
+
+    for id in 10 12 13 15; do
+        copr-cli delete-build $id
+    done
+    
+This has been no problem for small projects. But it was a big issue for projects with thousands of packages (e.g., python rebuilds). After each `delete-build` we run `createrepo_c` to update repository. And for these big projects, it can run for several minutes. It can easily take a day to delete hundreds of builds this way.
+
+Since the copr-cli-1.87 (going to be released this week), you are now able to delete a group of builds specified by their IDs with a single command in the following style:
```
Comment (FrostyX) [inline _posts/2020-06-01-deleting-list-of-builds.md:15]: I'd put `copr-cli-1.87` into backticks.

File: _posts/2020-06-01-deleting-list-of-builds.md [hunk 3]
```diff
@@ -4,8 +4,16 @@ author: Dominik Tureček
...
 
     copr-cli delete-build 10 12 13 15
 
-Doing this, not only is it faster than deleting each build separately and waiting until it's done, but the whole deletion is handled as a single action on the backend, hopefully leading to a shorter time needed to delete larger numbers of builds.
+This generates one action for the backend. Delete all the builds at once and it calls `createrepo_c` only once. This will leads to a much shorter time needed to delete larger numbers of builds.
```
Comment (FrostyX) [inline _posts/2020-06-01-deleting-list-of-builds.md:19]: This generates one action for the backend, deletes all the builds at once and it calls `createrepo_c` only once.

Comment (FrostyX) [reply _posts/2020-06-01-deleting-list-of-builds.md:19]: This leads to a much shorter time needed for deleting larger numbers of builds.

**Comment by xsuchy** [comment]
updated

File: _posts/2020-06-01-deleting-list-of-builds.md [hunk 4]
```diff
@@ -4,16 +4,16 @@ author: Dominik Tureček
 layout: post
 ---
 
-Previously, when you wanted to delete lots of build in the Copr you have to do something like:
+Previously, when you wanted to delete lots of build in the Copr, you have to do something like:
```
Comment (FrostyX) [inline _posts/2020-06-01-deleting-list-of-builds.md:7]: s/have/had

but that's the last thing.

**Comment by xsuchy** [comment]
updated

**Comment by praiskup** [comment]
+1, thank you

**Pull request merged**